dbms/src/Storages/KVStore/Decode/RegionTable.cpp

```diff
@@ -68,16 +63,18 @@ RegionTable::Table & RegionTable::getOrCreateTable(const KeyspaceID keyspace_id,
 RegionTable::InternalRegion & RegionTable::insertRegion(Table & table, const Region & region)
 {
     const auto range = region.getRange();
-    return insertRegion(table, *range, region.id());
+    return insertRegion(table, *range, region);
 }
```

Comment:
Make line 108 in `RegionTable::getOrInsertRegion` as `return insertRegion(table, *region.getRange(), region);` and remove this redundant function?

Reply:
OK